index.html

```diff
@@ -21,11 +23,12 @@
           </section>
      </nav>
   
-
   <div class="container">
     <section class="text1">
```

Comment:
- [ ] Nice work wrapping the the main title with a `h1` tag :clap: However it would be nice to remove this `section` tag from the code and give the `class="text1"` to the `h1` tag. That will make your code clean.

The `section` tag should use only for section in your page. [Read more about `section` tag](https://www.w3schools.com/tags/tag_section.asp)